Trappist/src/Promact.Trappist.Test/Tests/TestSettingsRepositoryTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.Repository.TestSettings;
+using Microsoft.Extensions.DependencyInjection;
+using System.Linq;
+using Promact.Trappist.Repository.Tests;
+using Xunit;
+using System.Threading.Tasks;
+
+namespace Promact.Trappist.Test.Tests
+{
+    [Collection("Register Dependency")]
+    public class TestSettingsRepositoryTest
+    {
+        private readonly Bootstrap _bootstrap;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the "_bootstrap" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)